.travis.yml

```diff
@@ -22,4 +22,4 @@ script:
   - spack load environment-modules
   - source <(spack module loads gotcha leveldb)
   - sh autogen.sh && ./configure && make
-  - ./scripts/checkpatch.sh
+  # - ./scripts/checkpatch.sh
```

Comment:
Did you mean to comment this out? Or is this an artifact of exhausting our travis-ci.com trial builds?

Reply:
It's a temporary thing to get past the false positive it's tripping on in the WRAP macros.  I suspect we'll also want to leave it off for the style PR.  Wanted to get those two through as one-offs after which we can reenable it.

Reply:
Well but we can always merge a PR even if some aspects fail the style check. But I'd still prefer to see the output in case it finds valid stuff. I'd expect it to complain about the // comments, for example.

Reply:
back on